test/Distributed/Runtime/distributed_actor_to_actor.swift

```diff
@@ -0,0 +1,72 @@
+// RUN: %empty-directory(%t)
+// RUN: %target-swift-frontend-emit-module -emit-module-path %t/FakeDistributedActorSystems.swiftmodule -module-name FakeDistributedActorSystems %S/../Inputs/FakeDistributedActorSystems.swift
+// RUN: %target-build-swift -module-name main %import-libdispatch -j2 -parse-as-library -Xfrontend -disable-availability-checking -I %t %s %S/../Inputs/FakeDistributedActorSystems.swift -g -o %t/a.out -enable-experimental-feature BuiltinModule
+// RUN: %target-codesign %t/a.out
+// RUN: %target-run %t/a.out | %FileCheck %s --color
+
+// REQUIRES: executable_test
+// REQUIRES: concurrency
+// REQUIRES: distributed
+// REQUIRES: libdispatch
+
+// rdar://76038845
+// UNSUPPORTED: use_os_stdlib
+// UNSUPPORTED: back_deployment_runtime
+
+// FIXME(distributed): Distributed actors currently have some issues on windows rdar://82593574
+// UNSUPPORTED: OS=windows-msvc
+
+import Builtin // FIXME: Part of a hack to get the protocol conformance defined properly
+import Dispatch
+import Distributed
+import FakeDistributedActorSystems
+
+typealias DefaultDistributedActorSystem = FakeRoundtripActorSystem
+
+extension DistributedActor {
+  func f() {
+    // FIXME: Part of a hack that forces the protocol conformance for DistributedActor -> Actor to be defined
+    Builtin.distributedActorAsAnyActor(self)
+  }
+}
+
+distributed actor Worker {
+  var counter: Int = 0
+
+  distributed func test() async {
+    print("Starting on distributed actor \(self)")
+    counter = counter + 1
+    let result = await asLocalActor.doSomethingAsync {
+      incrementCounter()
+      return "\(counter)"
+    }
+    print("Distributed actor received \(result)")
+  }
+
+  func incrementCounter() {
+    counter += 1
+  }
+}
+
+extension Actor {
```

Comment:
Oh that's more powerful than I had assumed even, so we can even do an extension like this now, sweet! That's quite interesting 👀 

(I don't see any holes in the model -- it'll work nicely, since extensions on Actor cannot be `distributed` so this works nicely, we can put methods on here and they'll be only accessible while isolated on a DA, that's great)